src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -64,22 +64,18 @@ function LtiConfigForm({ onSubmit, intl, formRef }) {
       <Form ref={formRef} onSubmit={handleSubmit}>
         <h3 className="mb-3">{providerName}</h3>
         <p>
-          {showLTIConfig ? (
-            intl.formatMessage(messages.formInstructions)
-          ) : (
-            <FormattedMessage
-              {...messages.adminOnlyConfig}
-              values={{
-                providerName,
-                platformName: getConfig().SITE_NAME,
-                supportEmail: supportEmail ? (
-                  <MailtoLink to={supportEmail}>{supportEmail}</MailtoLink>
-                ) : (
-                  'support'
-                ),
-              }}
-            />
-          )}
+          <FormattedMessage
```

Comment:
I think the expectation is to have both the messages for staff. The one below is as well
'Complete the fields below to set up your discussion tool.'

Reply:
@marcotuts Is that the expectation?

Reply:
adding this to the ticket as well. for visibility.